src/main/java/no/unit/nva/hamcrest/DoesNotHaveNullOrEmptyFields.java

```diff
@@ -106,9 +91,28 @@ private boolean isEmpty(PropertyValuePair mir) {
             } else if (mir.value instanceof String) {
                 String str = (String) mir.value;
                 return str.isBlank();
+            } else if (mir.value instanceof JsonNode) {
```

Comment:
Not from this PR, but what exactly does 'mir' mean?

Reply:
fixed